src/components/file-tree/file-tree.ts

```diff
@@ -0,0 +1,48 @@
+import { PluginObject } from 'vue';
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+
+import { ModulVue } from '../../utils/vue/vue';
+import { FILE_TREE_NAME } from '../component-names';
+import IconFilePlugin from '../icon-file/icon-file';
+import IconPlugin from '../icon/icon';
+import WithRender from './file-tree.html';
+
+const FOLDER_OPEN: string = 'm-svg__file-openoffice-math';
+const FOLDER_CLOSED: string = 'm-svg__file-zip';
+@WithRender
+@Component
+export class MFileTree<T> extends ModulVue {
+
+    @Prop()
+    file: T;
+
+    @Prop({ default: false })
+    isOpen: boolean;
+
+    internalFolderIcon: string = FOLDER_CLOSED;
+
+    get isAFolder(): boolean {
+        return !this.file['idNode'];
```

Comment:
return !this.file.idNode ou utilise hasOwnProperty :)

Reply:
Done